content/addons.md

```diff
@@ -1,14 +1,142 @@
 # Addons
 
-<!-- What is an addon, link to Ember Observer -->
+The modern web developer has an incredible selection of open source code that they can use through the npm ecosystem, plus great tools for writing their own modules. Ember developers are free to use regular npm packages in their apps, but there are also thousands of packages that are made specifically for the Ember ecosystem. We call such packages "addons."
 
-## Learning how to write an addon
+An addon can include everything from JavaScript code, compiling tools, deployment pipelines, templates, stylesheets, and more.
 
-<!-- Suggest looking at source code of other addons -->
+In this guide, we'll cover:
+- Finding and using community addons
+- Addon file structure
+- Writing your an addon within an app
+- Writing an addon that can be shared
+- Turning a regular npm package into an addon
+- Testing addons
+- Managing assets
+- and more
 
-## Addon file structure
+## Finding and using community addons
 
-<!-- what are different sections actually used for -->
+[Ember Observer](https://www.emberobserver.com/) is the definitive way to explore community addons. Although addons can be found by [searching the npm repository directly](https://www.npmjs.com/search?q=ember), Ember Observer has ranked lists of most popular addons and ratings to help developers choose between them. Most are made to drop right into your app with zero configuration. Many addons are backwards-compatible to earlier versions of Ember too!
+
+To install an addon, use:
+
+```bash
+ember install <addon-name>
+```
+
+To be safe, it's a good idea to restart the local server after installing new dependencies, and especially before trying to debug an addon that isn't working.
+
+The `install` command creates an entry in the app's `package.json`, and downloads the addon and its dependencies into `node_modules`. Some addons also create new files or make modifications when they are installed.
+
+### Choosing an addon
+
+The [top 100 list](https://www.emberobserver.com/lists/top-addons) is an important first stop for new developers. Many Ember users have a personal set list of addons that they include in all their apps, and this is the best way to find out what those addons might be. Some addons are totally unique to Ember, while others exist to make it easier to use features of regular npm packages within an app.
```

Comment:
I suggest to name dozen of addons and briefly descibe them. It will also serve as an explanation what is addon

here is a list:
### DSL for backend
ember-data - provides abstraction for . To get data you will write `this.store.query('artist', {filter: {name: 'Adele'},include: 'albums'})`
ember-apollo-client - allows to use Appolo Client and GraphQL

### Components
ember-bootstrap - Wrapper on bootstrap library
ember-paper - Material UI Components
ember-leaflet - Declarative mapping
ember-power-select - Rich component for selects
ember-power-calendar - Rich component for calendar

### Helpers 
ember-math-helpers
ember-composable-helpers

### Additional features
ember-concurrency
liquid-fire
ember-cp-validations
ember-decorators

### Authentication
ember-simple-auth

### Internationalization
ember-intl

### Development tools
ember-cli-mirage
ember-cli-page-object
ember-try
ember-cli-update
ember-a11y-testing
ember-cli-deploy
ember-cli-release

Also, I think for the description its enough to write 2-3 sentences (but not less).

I can finish the descriptions for all of them, if you agree with idea (besides feel free to add/remove some)

Reply:
@chilicoder As I look at your list, I wonder if we should actually be repairing some of the category tags in Ember Observer itself so that we can just link to them. I found that some things were unpredictably categorized. This might help people explore better and avoid "playing favorites." I had decided that the benefit to new users outweighed the risk, but maybe this is a third way. Thoughts?

Reply:
fwiw, Improving categories in ember observer is definitely helpful to cross link them from the ember.js guides as well.

Reply:
@jenweber oh.. I just realised that I never used categories in EmberObserver. 

I think its better not to spread on several tasks. By my proposal I just wanted to name some widely used addons to help beginners get into the topic. If you approve, I will just add better descriptions here. And I just made up categories in 5 minutes. Its not a big deal here, can be rearranged later. 

And if speak about emberobserver categories, we can initiate move to the better structure and when it will be finished update this manual with links and rearrangements. I just checked them..looks like too many categories, some of which overlap, some of which can be combined etc. If you think we need to initiate a clean up here... I can propose a draft for the new structure

Reply:
@chilicoder all good points. If you want to open a PR against this branch with similar info, I’ll accept it. Let’s see where it takes us! Thank you!